### PR TITLE
Assignment5

### PR DESCRIPTION
**Pull request by kushagra-successive**

### CODE REVIEW

File: Assignment5/src/utils/helperFunction.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const jwt = require("jsonwebtoken");
+
+const secretKey = "mysecret";
```
Comment (manish-successive) [inline Assignment5/src/utils/helperFunction.js]: Still I see secrets coming as hard coded. It should come form env file

File: Assignment5/src/controllers/asyncController.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+const createError = require("http-errors");
+const promise = async (req, res, next) => {
```
Comment (manish-successive) [inline Assignment5/src/controllers/asyncController.js]: Promise is not correct as per naming convention. Please choose relevant names.

**Review by manish-successive — CHANGES_REQUESTED** [review]
I do not see any .gitIgnore file. And I also see node_modules folder present in the repo. Please take care of this.

File: Assignment5/src/controllers/asyncController.js (hunk 2 above)
Comment (kushagra-successive) [reply Assignment5/src/controllers/asyncController.js]: Changes done

**Review by manish-successive — APPROVED** [review]

**Pull request merged**